app/cli/cmd/root.go

```diff
@@ -195,13 +195,13 @@ func loadControlplaneAuthToken(cmd *cobra.Command) (string, error) {
 	// If the CMD uses a robot account instead of the regular auth token we override it
 	// TODO: the attestation CLI should get split from this one
 	if _, ok := cmd.Annotations[useWorkflowRobotAccount]; ok {
-		if robotAccount != "" {
+		if attAPIToken != "" {
```

Comment:
I'd change this code now to just contain the failure statement if the attAPItoken==""

I see no much value In that debug message anymore.